src/kafka.rs

```diff
@@ -65,12 +65,18 @@ pub async fn consume_alerts(
     group_id: Option<String>,
     exit_on_eof: bool,
     max_in_queue: usize,
+    queue_name: Option<String>,
 ) -> Result<(), Box<dyn std::error::Error>> {
     let group_id = match group_id {
         Some(id) => id,
         None => uuid::Uuid::new_v4().to_string(),
     };
 
+    let queue_name = match queue_name {
+        Some(q) => q,
+        None => "ZTF_alerts_packets_queue".to_string(),
+    };
```

Comment:
We should keep an eye on this kind of special casing with optional args. It's probably fine for now, at some point we may want to make `queue_name` required and call it with `"ZTF_alerts_packets_queue"` when we need to.